### ISSUE TITLE
/IssueTrackerProjectId does nothing when Issue Tracker is Jira

### ISSUE THREAD

**Issue by neoKushan** [GitTools/GitReleaseNotes#94]
Unfortunately, this causes GitReleaseNotes to dump all closed issues from all projects.

Not sure if other issue trackers are supported, only tested with Jira.

**Comment by JakeGinnivan** [comment]
I noticed that.. I added a todo to the code to fix that and told Geert :)

Still lots to do on this project, just struggling for time to complete.

Sent from my Windows Phone
________________________________

**Comment by GeertvanHorrik** [comment]
Yes, it's a catch 22, we need the author of the atlassian sdk to accept a few pr before we can fix this.

**Comment by neoKushan** [comment]
I'm surprised the Atlassian SDK is missing such a basic feature.

**Comment by GeertvanHorrik** [comment]
Well, it has that feature using the SOAP interface, but Atlassian disabled that last month on all cloud instances. This means that the library has to implement it as REST as well. I wrote a lot of extension methods that solve all of this, but the maintainer has to accept it. Once that is done, we will continue working on this.
